src/utils/router_state.rs

```diff
@@ -379,7 +397,7 @@ pub fn get_possible_flights(
     latest_arrival_time: Option<Timestamp>,
     vehicles: Vec<Vehicle>,
     existing_flight_plans: Vec<FlightPlan>,
-) -> Result<Vec<FlightPlanData>, String> {
+) -> Result<Vec<(FlightPlanData, Vec<FlightPlanData>)>, String> {
```

Comment:
this function's getting long enough to break into helpers, like 
```rust
fn get_possible_flights( ... ) {
    // initialize common vars

    let Ok(result) = tier_one() {
        // return response
    }

    // where later tiers do things like deadhead calculations, etc.
}

Reply:
Yes, I'm planning on cleanup, once I have last scenario in